support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPageReducer.js

```diff
@@ -20,6 +21,7 @@ export type State = {
   common: CommonState,
   page: {
     productPrices: ?ProductPrices,
+    promotionCopy: ?PromotionCopy,
```

Comment:
I don't think promotion copy will change while the page is open.  does it really need to be copied in the state rather than leaving it as a global const?

Reply:
There's a lot of stuff in Redux state on the landing pages that won't change while the page is open- I think it's worth considering if we need to put any of it into Redux, but I think it would be more confusing currently to have this arbitrarily not happen for some properties or on some pages but do it on others.

Reply:
yeah agreed.  I tend not to put things in redux if I don't have to, but I can see the value of consistency at least in this change.